shelley/chain-and-ledger/executable-spec/test/Test/Shelley/Spec/Ledger/Address.hs

```diff
@@ -81,19 +84,69 @@ goldenTests =
         "addrEnterpriseS"
         putAddr
         (Addr scriptHash StakeRefNull)
-        "7205060708"
+        "7205060708",
+
+      -- serialisation golden tests
+      goldenSerialisation
+        "addrEnterpriseK for network id = 0"
```

Comment:
maybe it is not for network id = 0 ?

Reply:
indeed it's not, the concrete crypto `C` is setting it to `Offline` (which is 2).

Reply:
yes, I got it. You use ConcreteCrypto impl of Crypto interface. Added RealisticCrypto here -> https://github.com/input-output-hk/cardano-ledger-specs/pull/1443/commits/8c7a0d69f361f4195edd7ed89203813d7aac3d77